### PR TITLE
Fix trigger function

### PR DESCRIPTION
**Pull request by yoojinko**

### CODE REVIEW

File: src/controllers/appController.ts [hunk 1]
```diff
@@ -35,10 +35,30 @@ export default class AppController {
...
+    }
+    const depositValue = this.buildSetFunctionValue(depositParam);
+    const depositFuncOp = buildSetOperation("SET_FUNCTION", depositParam.ref, depositValue);
+    setFunctionOps.push(depositFuncOp);
+
+    const serviceFuncPath = Path.app(appName).request("$userAddress", "$requestKey")
+    const serviceFuncUrl = `${serviceUrl}/service`;
+    const serviceFuncParam: setTriggerFunctionParm = {
+      ref: serviceFuncPath,
+      function_id: "service-tregger",
```
Comment (akastercomcom) [inline src/controllers/appController.ts:54]: trigger?

File: src/controllers/appController.ts [hunk 2]
```diff
@@ -35,10 +35,30 @@ export default class AppController {
...
 
-    const depositParam = this.depositTriggerFunctionConfig(appName, serviceUrl);
-    const value = this.buildSetFunctionValue(depositParam);
-    const funcOp = buildSetOperation("SET_FUNCTION", depositParam.ref, value);
-    setFunctionOps.push(funcOp);
+    const depositPath = `${Path.app(appName).depositOfUser("$userAddress")}/$transferKey`
+    const depositUrl = `${serviceUrl}/deposit`;
+    const depositParam: setTriggerFunctionParm = {
+      ref: depositPath,
+      function_id: "deposit-tregger",
```
Comment (akastercomcom) [inline src/controllers/appController.ts:42]: ditto

**Review by akastercomcom — APPROVED** [review]
LGTM
Thanks!

File: src/controllers/appController.ts (hunk 1 above)
Comment (yoojinko) [reply src/controllers/appController.ts]: Done. Thanks!

File: src/controllers/appController.ts (hunk 2 above)
Comment (yoojinko) [reply src/controllers/appController.ts]: Done. Thanks!

**Comment by yoojinko** [comment]
Thanks for the review!

**Pull request merged**